setup/src/integration/java/org/openremote/setup/integration/KeycloakTestSetup.java

```diff
@@ -83,6 +85,8 @@ public KeycloakTestSetup(Container container) {
     public void onStart() throws Exception {
         super.onStart();
 
+        ManagerIdentityProvider identityProvider = identityService.getIdentityProvider();
+
         // Realms
         realmMaster = identityService.getIdentityProvider().getRealm(Constants.MASTER_REALM);
```

Comment:
Seems like a duplicate pattern; we should between
- Caching `identityProvider` in an variable, or..
- use `identityService.getIdentityProvider()` throughout the setup code.